README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"> <img height="100" src="https://raw.githubusercontent.com/twishasaraiya/GuessTheQuote/master/design-assets/icon.png"></p>
+<p align="center"> <img height="200" src="https://github.com/ManajitPal/guess-quote/blob/master/client/public/assets/png/logo-complete.png"></p>
```

Comment:
Can we switch to this GitHub link itself?

Reply:
ya.. bt that would be only possible after it is merged.